src/avatar.scss

```diff
@@ -0,0 +1,172 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+/*!
+.fd-avatar+((--circle), (--thumbnail), (--xxs|--xs|--s|--m|--l|--xl|--xxl))
+*/
+$block: #{$fd-namespace}-avatar;
+
+.#{$block} {
+  // LOCAL VARS (set all themeable properties, always include !default)
+  $fd-const-ratio: 1rem;
+  $fd-avatar-sizes: (
+    xs: $fd-const-ratio * 2,
+    s: $fd-const-ratio * 3,
+    m: $fd-const-ratio * 4,
+    l: $fd-const-ratio * 5,
+    xl: $fd-const-ratio * 7,
+  ) !default;
+  $fd-avatar-border-radius: $fd-border-radius !default;
+  $fd-avatar-border-fade: rgba(0, 0, 0, 0.5); // Should be fade(var(--sapContent_ShadowColor), 10);
+  $fd-avatar-color: fd-color("text", 5) !default;
+  $fd-avatar-background-color: fd-color("accent", 6) !default;
```

Comment:
I think we agreed not to use fd font/color mixins and functions and instead use documentation CSS variables like [here](https://github.com/SAP/fundamental-styles/blob/08c87ec8639765d82e28bf3f8da72d37ae6cc0dc/src/info-label.scss#L44).